packages/protocol/contracts/governance/Validators.sol

```diff
@@ -451,8 +371,8 @@ contract Validators is
    */
   function _updateValidatorScore(address validator, uint256 uptime) internal {
     address account = getLockedGold().getAccountFromValidator(validator);
-    require(isValidator(account), "isvalidator");
-    require(uptime <= FixidityLib.fixed1().unwrap(), "uptime");
+    require(isValidator(account));
+    require(uptime <= FixidityLib.fixed1().unwrap());
```

Comment:
I'm not sure if I asked this before, but do we not provide these strings? In my editor it is actually a lint violation